docs/edgeql/ddl/links.rst

```diff
@@ -80,6 +82,11 @@ Parameters
     the *required* attribute, i.e it is not possible to make a
     required link non-required by extending it.
 
+:eql:synopsis:`OPTIONAL`
+    If specified, the link is considered *optional* for the parent
```

Comment:
OPTIONAL is default, so it doesn't matter if it's specified or not. Please reflect that it's the default.

Reply:
Updated to clarify it's the default.